src/scripts/main.js

```diff
@@ -1,3 +1,41 @@
 'use strict';
 
-// write your code here
+const url
+= 'https://mate-academy.github.io/phone-catalogue-static/api/phones.json';
+
+const controller = new AbortController();
```

Comment:
instead of using `AbortController`, you could have used `Promise.reject` method inside `setTimeout` call
it could be something like this
```
function getPhones() {
  return new Promise((resolve, reject) => {
    fetch(url)
      .then(response => resolve(response.json()));

    setTimeout(() => {
      reject(new Error('something wrong'));
    }, 5000);
  });
};
```